jest.config.js

```diff
@@ -2,6 +2,7 @@ module.exports = {
   collectCoverageFrom: [
     '<rootDir>/app/scripts/constants/error-utils.js',
     '<rootDir>/app/scripts/controllers/network/**/*.js',
+    '<rootDir>/app/scripts/controllers/network/**/*.ts',
```

Comment:
We should make the equivalent change to the Mocha configuration as well, so it's not counted towards coverage there.

Reply:
Where would I need to make this change?

Reply:
The file is `.mocharc.js`, you should see a list there similar to this one

Reply:
Ah, thanks. Like this? [0b0c576](https://github.com/MetaMask/metamask-extension/pull/18476/commits/0b0c5765f69e72dc0ce7c6fcb7b4fa539f99b2c7)

Reply:
So that mocha config is a list of test suites that are run, not files to gather coverage from. I was suggesting adding `<rootDir>/app/scripts/controllers/network/**/*test.ts` to that list. However I think I was wrong.

I had suggested that it needed an update because I assumed that some of our Mocha tests were TypeScript, but [apparently not](https://github.com/MetaMask/metamask-extension/blob/develop/test/run-unit-tests.js#L74). Sorry about that, that change looks to be unnecessary after all.

Reply:
Cool. No problem! Reverted.